eyieko/landville-backend-web-api
# 168073038 create seed data scripts

## Title
Create seed data script

## Description

As a developer, I want to run a script that prepopulates the development database so that I won't have to manually add test data to the database.

## Type of change
Please select the relevant option


How Has This Been Tested?
Please describe the tests that you ran to verify your changes


Checklist:
  My code follows the style guidelines of this project


### How can this be manually tested?
1. Git fetch my branch. `git fetch origin ft-create-seed-data-script-168073038`
2. Activate the `virtualenv`
3. Run `python load_fixtures.py` to install the fixtures
4. When you check your database there should be some sample data loaded from the fixtures.

Related Pivotal Tracker stories
[#168073038](https://www.pivotaltracker.com/story/show/168073038)
